src/main/java/OldCode/Duke.java

```diff
@@ -0,0 +1,85 @@
+package OldCode;
+import java.util.Scanner;
+
+public class Duke {
+
+    public static void main(String[] args) {
+
+        printMessage message = new printMessage();
+        message.printIntro();
+        Information input = new Information();
+        Scanner myObj = new Scanner(System.in);
+        boolean condition = true;
+        int counter = 0;
+
+
+        while (condition) {
+            String userInput = myObj.nextLine();
+            String instruction = userInput.contains(" ") ? userInput.substring(0, userInput.indexOf(" ")): userInput;
+            String instructionTask = userInput.contains(" ") ? userInput.substring(userInput.indexOf(" ") + 1) : userInput;
+
+            switch (instruction) {
+            case "bye":
+                System.out.println("widePeepoSad :(" + "\n" + "_______________________________________________________\n");
+                condition = false;
+                break;
+            case "list":
+                for (int i = 0; i < 100; i++) {
+                    if (input.List[i] == null) {
+                        break;
+                    } else {
+                        System.out.println((i + 1) + ".[" + input.List[i].getTaskIcon() + "]" + "[" + input.List[i].getStatusIcon() + "] " + input.List[i].getDescription());
```

Comment:
Line is too long. Consider wrapping the line into multiple lines (more than 120 characters).